packages/abstractions/src/serialization/kiotaJsonSerializer.ts

```diff
@@ -8,64 +8,72 @@ import { deserialize, deserializeCollection, serialize, serializeCollection, ser
 import type { Parsable } from "./parsable";
 import type { ParsableFactory } from "./parsableFactory";
 import type { ModelSerializerFunction } from "./serializationFunctionTypes";
+import { SerializationWriterFactoryRegistry } from "./serializationWriterFactoryRegistry";
+import type { ParseNodeFactory } from "./parseNodeFactory";
 
 const jsonContentType = "application/json";
 /**
  * Serializes a parsable object into a buffer
+ * @param serializationWriterFactoryRegistry the serialization writer factory registry
  * @param value the value to serialize
  * @param serializationFunction the serialization function for the model type
  * @returns a buffer containing the serialized value
  */
-export function serializeToJson<T extends Parsable>(value: T, serializationFunction: ModelSerializerFunction<T>): ArrayBuffer {
-	return serialize(jsonContentType, value, serializationFunction);
+export function serializeToJson<T extends Parsable>(serializationWriterFactoryRegistry: SerializationWriterFactoryRegistry, value: T, serializationFunction: ModelSerializerFunction<T>): ArrayBuffer {
```

Comment:
the goal of those methods was to make it easy to serialize/deserialize.
Since we need an instance, what do you think about making them instance methods of the `SerializationWriterFactoryRegistry` (and equivalent) classes instead?